ls1intum/Artemis
# `Assessment`: Show disabled correction rounds in assessment overview

### Checklist
#### General


#### Client

### Motivation and Context


Some testers and tutors were confused about the second assessment round not showing up in the assessment page.

### Description

The assessment page now shows all assessment rounds that will be made during the assessment.
To reduce the overall size of the UI, the explanation about not being able to assess the same submission in different assessment rounds is moved to a tool tip.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Exam with a second assessment round

1. Log in to Artemis
2. Participate in the exam as the student
3. Go to the assessment board after the deadline of the exam and check the UI of the assessment page for the different exercises
4. Enable/Disable the second correction round and check for confusing UI

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


**Old:**
Second Round disabled:
![Bildschirmfoto 2022-07-28 um 19 08 52](https://user-images.githubusercontent.com/38322605/181603104-6d0bdc7f-f15e-40b1-b992-23ea783a18a3.png)
Second Round enabled:
![Bildschirmfoto 2022-07-28 um 19 09 04](https://user-images.githubusercontent.com/38322605/181603153-df2cc6e7-befa-473a-92bd-4695c96f3c01.png)

**New:**
Second Round disabled:
![Bildschirmfoto 2022-07-28 um 19 10 39](https://user-images.githubusercontent.com/38322605/181603231-19a64344-488d-457c-bb1c-9ebf1c6cc407.png)
Second Round enabled:
![Bildschirmfoto 2022-07-28 um 20 11 45](https://user-images.githubusercontent.com/38322605/181608039-99133e39-3c21-41bc-a3fb-17c900052d78.png)
